content/react/pt/get-started.md

```diff
@@ -44,34 +44,34 @@ yarn start
 ```
 
 <div class="aside">
-  Nota: Se <code>yarn test</code> emitir um erro, será necessário instalar o pacote <code>watchman</code>, tal como recomendado neste <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">problema</a>
+  Nota: Se o comando <code>yarn test</code> produzir um erro, será necessário instalar o pacote <code>watchman</code>, conforme recomendado nesta <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">thread</a>
 </div>
 
 Sendo estes os seguintes: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
 
 ![3 modalidades](/app-three-modalities.png)
 
 
-Dependendo de qual parte da aplicação que estamos a trabalhar, podemos querer executar um ou mais simultâneamente.
-Visto que neste caso, o foco é a criação de um componente de interface de utilizador simples, iremos cingir-nos somente á execução de Storybook.
+Dependendo de qual parte da aplicação que estamos trabalhando, podemos querer executar um ou mais simultâneamente.
+Visto que, neste caso, o foco é a criação de um componente de interface de usuário simples, continuaremos executando o Storybook.
 
 ## Reutilização CSS
 
 A Taskbox reutiliza elementos de design do tutorial de React e GraphQL
-[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), como tal não será necessária a criação de CSS neste tutorial. 
+[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), por isso não será necessária a criação de CSS neste tutorial. 
 
-Com isto o conteúdo do ficheiro LESS será compilado num único ficheiro CSS e incluido na aplicação.
+Desta forma o conteúdo do arquivo LESS será compilado em um único arquivo CSS e incluído na aplicação.
```

Comment:
"Arquivo" and "ficheiro" are one and the same, "num" and "em um" are the one and the same, both readers can understand it, why the change? 
Thanks for the accent, missed that one